README.md

```diff
@@ -1,20 +1,40 @@
 # Demo
 
-To start your Phoenix server:
+## Spec
 
-  * Install dependencies with `mix deps.get`
-  * Create and migrate your database with `mix ecto.setup`
-  * Install Node.js dependencies with `npm install --prefix assets`
-  * Start Phoenix endpoint with `mix phx.server`
+We will two database tables: "users" and "users_tokens":
```

Comment:
```suggestion
We will have two database tables: "users" and "users_tokens":
```

Reply:
Thanks! Btw, just to be clear, the README won't be part of code generators. :)